src/test/java/net/datafaker/service/FakeValuesServiceTest.java

```diff
@@ -149,60 +151,33 @@ void resolveKeyToPropertyWithAPropertyWithoutAnObject() {
         verify(dummy).hello();
     }
 
-    @Test
-    void resolveKeyToPropertyWithAPropertyWithAnObject() {
```

Comment:
mocked tests is painful to maintain
at the same time there are several normal (non mocked tests) covered this functionality
e.g. tests with `InsectFromFile`